lib/usd/utils/MergePrims.cpp

```diff
@@ -246,6 +247,46 @@ bool isLocalTransformModified(const UsdPrim& srcPrim, const UsdPrim& dstPrim)
     return false;
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+// Special normal attributes handling.
+//
+// The goal is to detect that a prim normals has *not* changed even though its representation
+// might have changed. The reason is that normals can be kept in the 'normals' attribute or in
+// the 'primvars:normals' attribute, with the latter having priority. Maya export to USD always
+// generates the 'normals' attribute, which would get hidden when merged. We now detect and correct
+// that situation.
+//----------------------------------------------------------------------------------------------------------------------
+
+const char normalsTokenName[] = "normals";
+const char pvNormalsTokenName[] = "primvars:normals";
+const char pvNormalsIndicesTokenName[] = "primvars:normals:indices";
+
+bool isNormalsProperty(const TfToken& name)
+{
+    // Note: tokens are interned to be unique and thus we don't want
+    //       their constructor to be global in order to avoid the
+    //       unspecified order of global construction.
+    //
+    //       That is why we declare them as static inside the functions.
+    static const TfToken normalsToken(normalsTokenName);
+    static const TfToken pvNormalsToken(pvNormalsTokenName);
```

Comment:
Normally we do this using the TF_DEFINE_PRIVATE_TOKENS macro. You can see an example: https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/render/vp2RenderDelegate/material.cpp#L119